### ISSUE TITLE
Stream support?

### ISSUE THREAD

**Issue by pfischermx** [yzprofile/ngx_http_dyups_module#107]
More of a 'feature request', but does this module already supports upstream streams (for TCP only)? Or you know of any existing forks that have this?

**Comment by yzprofile** [comment]
Not support stream upstream, no time to do that. 

welcome pull request. : )

**Comment by chobits** [comment]
If we want to support stream module, we should implement a new module for dynamic stream upstream, like ngx_stream_dyups_module.

Current dyups module is based on __http__ upstream module.
For nginx.conf, it can only work under `http {}` block.

**Comment by pfischermx** [comment]
Yeah I agree. I wish I still remembered C otherwise I'd provide a patch :/

**Comment by mehulmpt** [comment]
Just wondering if there is any update on this?
